go/client/cmd_team.go

```diff
@@ -11,9 +11,8 @@ import (
 
 func NewCmdTeam(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 	return cli.Command{
-		Name: "team",
-		// Hide it:
-		// Usage:        "Manage teams.",
+		Name:         "team",
+		Usage:        "Manage teams.",
```

Comment:
one small detail: the other usages don't have a period at the end, do they?  

anyway this all looks good to me.

Reply:
ok.

We should standardize, because once you go in a level, it's all periods...

```> kbcl -d=0 team help
NAME:
   keybase team - Manage teams.

USAGE:
   keybase team <command> [arguments...]

COMMANDS:
   create				Create a team or a subteam.
   add-member				Add a user to a team.
   remove-member			Remove a user from a team.
   edit-member				Change a user's role on a team.
   list-memberships, list-members	List your teams, or people on a team.
   show-tree				Show a team's subteams.
   rename				Change the name of a subteam.
   request-access			Request access to a team.
   list-requests			List requests to join teams.
   ignore-request			Ignore request to join a team.
   accept-invite			Accept a team email invitation.
   leave				Leave a team.
   delete				Delete a team.```

Reply:
hah, agreed, well, doesn't matter to me which way we choose